server/api/service/email/planDeposit.mustache

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html dir="rtl">
+
+<head>
+  <title>
+    {{#data.PLAN_COUNTY_NAME}} תכנית הופקדה להתנגדויות ב{{{data.PLAN_COUNTY_NAME}}} {{/data.PLAN_COUNTY_NAME}} {{^data.PLAN_COUNTY_NAME}} תכנית
+    הופקדה להתנגדויות באזור העניין שלך {{/data.PLAN_COUNTY_NAME}} - {{{data.PL_NAME}}}</title>
+</head>
+
+<body dir="rtl">
+  <h2 class="headline">יש לך התראה חדשה!</h2>
+  {{#data.PLAN_COUNTY_NAME}}
+    <h2 class="headline">תכנית הופקדה להתנגדויות ב{{data.PLAN_COUNTY_NAME}}</h2>
+  {{/data.PLAN_COUNTY_NAME}} {{^data.PLAN_COUNTY_NAME}}
+    <h2 class="headline">תכנית הופקדה להתנגדויות באזור העניין שלך</h2>
```

Comment:
@drorgivton What should be the text here?
@drorgivton consider that users are alerted of plans they have subscribed to, so it's not אזור ענין but תוכנית במעקב or something like that

Reply:
@GalGend @yaelal 

״תכנית במעקב הופקדה להתנגדויות״